contents/docs/cdp/batch-exports/index.md

```diff
@@ -32,12 +33,50 @@ A batch export is executed in batch runs depending on the configured frequency.
 
 As an example, creating a batch export of events with daily frequency today will schedule the first batch run to start right as tomorrow begins. Thus, the data exported are events that PostHog received from 00:00:00 until 23:59:59 of today.
 
-Batch runs are configured on the [exports page](https://app.posthog.com/exports) in your PostHog instance.
-
 > **Note:** When deciding which data falls within the bounds of a specific batch run, we look at the time when the data landed in PostHog's ClickHouse database. This means that network and processing delays can make an event that was sent within the bounds of a batch run fall in a future run.
 
-## How does this differ from the old export system?
+### Tracking progress
+
+On each batch export view we will be presented with a list of the latest runs executed:
+
+![batch export runs](../../../images/docs/batch-exports/batch-exports-runs.png)
+
+Each run has an indicator of the state (which can be either "Starting", "Running", "Failed", or "Completed"), the boundaries for the data exported (data interval start and data interval end), when the run actually started, and the option of retrying a specific run.
+
+
+## Historical exports
+
+Batch exports can export data that landed in PostHog before the batch export was created, known as historical data. For this, no new batch export has to be created: A batch export already knows the destination where we wish to export historical data, it only requires the boundaries for the historical export, in other words a start and an end date.
+
+A "Create historic export" button can be found in the UI:
+
+![batch exports ui](../../../images/docs/batch-exports/batch-exports-ui.png)
+
+Which will let users input the start and end date of the historical export:
+
+![create historic export](../../../images/docs/batch-exports/create-historic-export.png)
+
+Immediately afterwards, the historical export runs that fall within the bounds selected will be scheduled.
+
+> **Note:** A historical export does not check if the data already exists in the destination. Doing so would negatively impact the performance of the batch export, and potentially require more permissions on the user's database or storage. Moreover, we can never be sure if the data was moved somewhere else. Instead, we assume that users who request a historic export want all their historic data, which means that multiple historical exports over the same time period will produce duplicates.
+
+> **Note:** A batch export may optionally be created with an end date: Batch exports will never export data past this end date, even if requesting a historical export which exceeds this upper bound.
```

Comment:
We should change the implementation in a way that this is logical and intuitive to the user instead. Specifically 

> even if requesting a historical export which exceeds this upper bound.

There are only 2 reasonable user stories I can think of:
1. create the export and start exporting all events from this point forward (additionally one can trigger historical backfills for previous data)
2. create the export but don't start exporting events as the user only wants to trigger some historical backfills (potentially now and potentially other ones in the future - which isn't intuitive & why this note exists 🤷‍♀️ )

So to me the logical setting for the users is a boolean: live/continous events exporting enabled or not.
Separately the user can then trigger historical exports.

I find it super unintuitive to need to set some end date if I'm only looking to manually trigger exports, especially since depending on how our implementation works and maybe we had an issue with lag, then there could be impact to users. Imagine the user wants to trigger a historical backfill for events until yesterday and they want them to be delivered in order oldest to newest, now if we had an issue and a 2 day delay on batch exports, then once we fix the issue we'll trigger yesterday's events to be sent immediately and then get duplicates from the manually triggered batch exports (or in any case the ordering is wrong).

See also discussion in thread: https://posthog.slack.com/archives/C0374DA782U/p1692712916800799?thread_ts=1692712505.707579&cid=C0374DA782U

Reply:
cc @MarconLP  and @xvello for your thoughts on this

Reply:
> Imagine the user wants to trigger a historical backfill for events until yesterday and they want them to be delivered in order oldest to newest

We do not support any kind of special ordering when historically backfilling. I don't know why you are bringing up an example with a feature that doesn't exist. If you wish to propose this feature, I'm happy to consider it and eventually re-visit these docs, but I don't see how this makes any sense in docs for current feature set, and given our current priorities I doubt this would be worked on any time soon.

> (or in any case the ordering is wrong).

We do not guarantee any ordering so this is not an issue.

> then once we fix the issue we'll trigger yesterday's events to be sent immediately and then get duplicates from the manually triggered batch exports (or in any case the ordering is wrong).

Incorrect: If the user configured the batch export until yesterday, nothing will be triggered by us upon recovery. There is no duplicate problem.

Reply:
> 2. make sure that it's paused

Not necessarily, doing both this and 3. have the same effect.

> 3. set the end-date after our export date

Why? Just set it to the end of the period you are looking to historically export. Or not set it all and do 2. instead.

> 4. trigger historical export with it's own start and end date.

End date is pre-populated with today's date, but we should extend this in the frontend so that it instead uses the end date set in 3 (if any). This way only the start date is required.

> Why is that? What does the batch export end date have anything to do with the historical export ending date?

Because the end date of a Temporal Schedule applies to all runs, historical or not. No runs past the end date will be scheduled. Even if you do request a historical export beyond the end date, only runs until the end date will be scheduled.

Reply:
We could communicate better that there is no "historical export", there is only a batch export: A historical export is not a separate export, it's an operation you do on an existing batch export. Perhaps the frontend should say "Export historical data" instead of "Create historic export".

Reply:
> Incorrect: If the user configured the batch export until yesterday, nothing will be triggered by us upon recovery. There is no duplicate problem.

why?

> We do not guarantee any ordering so this is not an issue.

The old exporting did afaik and this is currently very much what replicator relies on because person property updates are at ingestion time.

For the purpose of this PR - let's just merge as is and we can discuss this part during the offsite